### PR TITLE
Create movie detail page with trailer, information and possibility to post and watch reviews

### PR DESCRIPTION
**Pull request by PontusVallgren**

### CODE REVIEW

**Comment by kaulfield23** [comment]
I just tested a bit, but got this error msg for movie Morbius, The batman and spiderman 
image
other movies works well except for these

**Comment by PontusVallgren** [comment]
> I just tested a bit, but got this error msg for movie Morbius, The batman and spiderman image other movies works well except for these

I think backgroundImg and trailer are missing on the 2 new movies in the db, batman and modus

**Comment by kaulfield23** [comment]
image
and also uuid is missing @type/uuid smth, so I can't run npm run build

**Comment by PontusVallgren** [comment]
> image and also uuid is missing @type/uuid smth, so I can't run npm run build

Added types

**Comment by kaulfield23** [comment]
image
This one has no style, I noticed that material ui can override css, so I took away all the css that i wrote for className with MUI, I see that only that button has problem with css. This is not important thing, but Boss baby's movie trailer won't show any video. I guess there is no video in youtube? anyway tested out leaving review as well, great! it works really well! 🥇

File: package.json [hunk 1]
```diff
@@ -16,14 +16,17 @@
...
     "@mui/material": "^5.6.2",
+    "@types/uuid": "^8.3.4",
     "bcrypt": "^5.0.1",
     "cookies": "^0.8.0",
     "mongoose": "^6.3.1",
     "next": "12.1.5",
     "react": "18.0.0",
     "react-dom": "18.0.0",
+    "tss-react": "^3.6.2",
+    "uuid": "^8.3.2"
```
Comment (kaulfield23) [inline package.json:27]: this one missing ','

File: src/components/FeedBackForm.tsx [hunk 2]
```diff
@@ -9,49 +9,52 @@ export default function FeedBackForm() {
     <>
       <h2>Övriga synpunkter eller frågor? Lämna gärna ett meddelande.</h2>
       <form
-        action="https://formsubmit.co/83afa7b9f826648c2534bbfc2f3f3571"
-        method="POST"
+
+        action='https://formsubmit.co/83afa7b9f826648c2534bbfc2f3f3571'
+        method='POST'
+        onSubmit={handleFormSubmit}
```
Comment (kaulfield23) [inline src/components/FeedBackForm.tsx:15]: it is missing handleFormSubmit function as well

**Comment by kaulfield23** [comment]
just checked, and it still placed like this with build like before.
image
it is because something override css like i wrote before. but since it is only this btn has css problem so I recommend u just give classname to div instead of button. I found a way that you could use classname to mui things but i'm too lazy to fix it haha

**Comment by kaulfield23** [comment]
image
I just tested, with this and justify-content center would work.

**Comment by PontusVallgren** [comment]
> just checked, and it still placed like this with build like before. image it is because something override css like i wrote before. but since it is only this btn has css problem so I recommend u just give classname to div instead of button. I found a way that you could use classname to mui things but i'm too lazy to fix it haha

Oh right, didn't really understand you at first. Hopefully its working now!

**Comment by kaulfield23** [comment]
snyggt! det ser bra ut 🥇

**Pull request merged**